patches/third_party-blink-renderer-modules-quota-storage_manager.cc.patch

```diff
@@ -0,0 +1,13 @@
+diff --git a/third_party/blink/renderer/modules/quota/storage_manager.cc b/third_party/blink/renderer/modules/quota/storage_manager.cc
+index ca889ad4ae2f4318071c91803ac26fb003e47a5e..58fbf4740df68c3ddfce97617231678fbe1655fb 100644
+--- a/third_party/blink/renderer/modules/quota/storage_manager.cc
++++ b/third_party/blink/renderer/modules/quota/storage_manager.cc
+@@ -55,7 +55,7 @@ void QueryStorageUsageAndQuotaCallback(
+ 
+   StorageEstimate* estimate = StorageEstimate::Create();
+   estimate->setUsage(usage_in_bytes);
+-  estimate->setQuota(quota_in_bytes);
++  estimate->setQuota(2147483648);
```

Comment:
Not sure if this would work, but maybe we can use chromium_src override that will redefine StorageEstimate as BraveStorageEstimate, where BraveStorageEstimate would subclass StorageEstimate and hide base's setQuota method with one that just sets the 2GB value?

So the override would look something like:
<pre>
#include "third_party/blink/renderer/modules/quota/storage_estimate.h"
#include "third_party/blink/renderer/bindings/modules/v8/v8_storage_estimate.h" 

namespace blink {
class BraveStorageEstimate : public StorageEstimage {
public:
...
  void setQuota(uint64_t quota) {
    quota = 2147483648;
    StorageEstimage::setQuota(quota);
  }
};
}  // namespace blink

#define StorageEstimage BraveStorageEstimate
#include "..../storage_manager.cc"
#undef StorageEstimage 
</pre>

Or something like that...

Reply:
@mkarolin I just tried the below override at `chromium_src/third_party/blink/renderer/modules/quota/storage_manager.cc` - it doesn't appear to be working though, I'm not getting 2GB anymore.

```cpp
#include "third_party/blink/renderer/modules/quota/storage_estimate.h"
#include "third_party/blink/renderer/bindings/modules/v8/v8_storage_estimate.h" 

namespace blink {
class BraveStorageEstimate : public StorageEstimate {
public:
  void setQuota(uint64_t quota) {
    quota = 2147483648;
    StorageEstimate::setQuota(quota);
  }
};
}  // namespace blink

#define StorageEstimage BraveStorageEstimate
#include "../../../../../../../third_party/blink/renderer/modules/quota/storage_manager.cc"
#undef StorageEstimage 
```

Reply:
It would also need
<pre>
static BraveStorageEstimate* Create() {
  return MakeGarbageCollected<BraveStorageEstimate>();
}
</pre>

Reply:
added that, didn't appear to make any difference